code/obj/artifacts/artifact_items/talisman.dm

```diff
@@ -0,0 +1,138 @@
+-Random movement speed increase from 0 - 5% - you feel drafts of air surrounding this artifact/you feel air moving quickly around this artifact
+*/
+
+/obj/item/artifact/talisman
+	name = "artifact talisman"
+	associated_datum = /datum/artifact/talisman
+	var/associated_effect
+	// swiftness vars
+	var/swiftness_mod
+	// fortune vars
+	var/money_amt
+	// spacefaring vars
+	var/space_prot
+	// elements vars
+	var/heat_prot
+	var/cold_prot
+	// strength vars
+	var/extra_hp
+	// protection vars
+	var/brute_prot
+	var/burn_prot
+	var/tox_prot
+
+	attack_self(mob/user)
+		. = ..()
+		if (!src.artifact.activated)
+			return
+		var/msg
+		switch(src.associated_effect)
+			if (SWIFTNESS)
+				msg = "You feel air drafts around [src]."
+			if (FORTUNE)
+				msg = "You feel lucky somehow."
+			if (SPACEFARING)
+				msg = "[src] feels VERY cold!!!"
+			if (ELEMENTS)
+				msg = "[src] feels warm and cold in different spots. [prob(99) ? null : "Sort of like that honk-pocket you once had..."]"
+			if (STRENGTH)
+				msg = "Holding [src] makes you feel strong."
+			if (PROTECTION)
+				msg = "You feel safe holding [src]."
+
+		boutput(user, SPAN_NOTICE(msg))
+
+	equipped(mob/user)
+		..()
+		src.add_effect_to_user(user)
+
+	unequipped(mob/user)
+		src.remove_effect_from_user(user)
+		..()
+
+	//pick_up_by(mob/M)
+	//	..()
+	//	if (src.loc == M)
+	//		src.add_effect_to_user(M)
+
+	proc/add_effect_to_user(mob/user)
+		switch(src.associated_effect)
+			if (SWIFTNESS)
+				APPLY_MOVEMENT_MODIFIER(user, /datum/movement_modifier/artifact_talisman_swiftness, src)
+			if (FORTUNE)
+			if (SPACEFARING)
+			if (ELEMENTS)
+				APPLY_ATOM_PROPERTY(current_user, PROP_MOB_COLDPROT, src, 100)
+			if (STRENGTH)
+				user.changeStatus("talisman_extra_hp", null, src.extra_hp)
+			if (PROTECTION)
+
+	proc/remove_effect_from_user(mob/user)
+		switch(src.associated_effect)
+			if (SWIFTNESS)
+				REMOVE_MOVEMENT_MODIFIER(user, /datum/movement_modifier/artifact_talisman_swiftness, src)
+			if (FORTUNE)
+
+			if (SPACEFARING)
+			if (ELEMENTS)
+			if (STRENGTH)
+				user.delStatus("talisman_extra_hp")
+			if (PROTECTION)
+
+	proc/select_effect()
+		APPLY_ATOM_PROPERTY(current_user, PROP_MOB_COLDPROT, src, 100)
+		src.associated_effect = pick(list(SWIFTNESS, FORTUNE, SPACEFARING, ELEMENTS, STRENGTH, PROTECTION))
+		switch(src.associated_effect)
+			if (SWIFTNESS)
+				src.swiftness_mod = rand(1, 5) / 100
+			if (FORTUNE)
+				src.money_amt = rand(100, 500)
+			if (SPACEFARING)
+				src.space_prot = rand(25, 75)
+			if (ELEMENTS)
+				src.heat_prot = rand(1, 25)
+				src.cold_prot = rand(1, 25)
+			if (STRENGTH)
+				src.extra_hp = rand(10, 50)
+			if (PROTECTION)
+				src.brute_prot = rand(1, 10)
+				src.burn_prot = rand(1, 10)
+				src.tox_prot = rand(1, 10)
+
+
+
+/datum/artifact/talisman
+	associated_object = /obj/item/artifact/talisman
+	type_name = "Talisman"
+	type_size = ARTIFACT_SIZE_MEDIUM
```

Comment:
Makes more sense for this to be `ARTIFACT_SIZE_TINY` instead, in my mind. We also have like 10 medium-sized artifacts and only 2 tiny-sized.

Reply:
Tiny size is used for the power cells, so would need to use those sprites. May do, will think about it a bit more

Reply:
Yeah, and I think those tiny sprites work better for a "talisman". Protective charms are usually small items, rocks or gems, etc.

Reply:
Added!